diff -up protobuf-30.1/cmake/protobuf-lite.pc.cmake.2~ protobuf-30.1/cmake/protobuf-lite.pc.cmake
--- protobuf-30.1/cmake/protobuf-lite.pc.cmake.2~	2025-03-13 20:50:35.000000000 +0100
+++ protobuf-30.1/cmake/protobuf-lite.pc.cmake	2025-03-14 10:21:52.292914803 +0100
@@ -1,12 +1,11 @@
 prefix=@CMAKE_INSTALL_PREFIX@
 exec_prefix=@CMAKE_INSTALL_PREFIX@
 libdir=@CMAKE_INSTALL_FULL_LIBDIR@
-includedir=@CMAKE_INSTALL_FULL_INCLUDEDIR@
 
 Name: Protocol Buffers
 Description: Google's Data Interchange Format
 Version: @protobuf_VERSION@
 Requires: @_protobuf_PC_REQUIRES@
-Libs: -L${libdir} -lprotobuf-lite @CMAKE_THREAD_LIBS_INIT@
-Cflags: -I${includedir} @_protobuf_PC_CFLAGS@
+Libs: -lprotobuf-lite @CMAKE_THREAD_LIBS_INIT@
+Cflags: @_protobuf_PC_CFLAGS@
 Conflicts: protobuf
diff -up protobuf-30.1/cmake/protobuf.pc.cmake.2~ protobuf-30.1/cmake/protobuf.pc.cmake
--- protobuf-30.1/cmake/protobuf.pc.cmake.2~	2025-03-14 10:21:52.292985978 +0100
+++ protobuf-30.1/cmake/protobuf.pc.cmake	2025-03-14 10:22:27.443455567 +0100
@@ -1,12 +1,11 @@
 prefix=@CMAKE_INSTALL_PREFIX@
 exec_prefix=@CMAKE_INSTALL_PREFIX@
 libdir=@CMAKE_INSTALL_FULL_LIBDIR@
-includedir=@CMAKE_INSTALL_FULL_INCLUDEDIR@
 
 Name: Protocol Buffers
 Description: Google's Data Interchange Format
 Version: @protobuf_VERSION@
 Requires: @_protobuf_PC_REQUIRES@
-Libs: -L${libdir} -lprotobuf@protobuf_LIBRARY_POSTFIX@ @CMAKE_THREAD_LIBS_INIT@
-Cflags: -I${includedir} @_protobuf_PC_CFLAGS@
+Libs: -lprotobuf@protobuf_LIBRARY_POSTFIX@ @CMAKE_THREAD_LIBS_INIT@
+Cflags: @_protobuf_PC_CFLAGS@
 Conflicts: protobuf-lite
